### PR TITLE
[#21][UI] As a user, I can see survey outro

### LINKED ISSUE #21: [UI] As a user, I can see survey outro
## Why
The application supports various types of survey items. After the user fills in all required survey items, the survey can be submitted and mark the survey as completed.

## Acceptance Criteria
- Display SVG animation after survey submission.
image

- After 2 seconds, redirect the user to the survey dashboard page.

### PR DESCRIPTION
**Pull request by hanam1ni**
- Closes #21 

## What happened 👀

- Add survey outro and play outro animation
- Add `jest-canvas-mock`, allowing Jest to run a test on pages that contain `canvas`. ([More detailed](https://tnodes.medium.com/setting-up-jest-with-react-and-phaser-422b174ec87e#:~:text=That%20null%20is%20actually%20supposed%20to%20be%20the%20canvas%20element.%20Jest%20uses%20jsdom%20as%20a%20psuedo%20DOM%2C%20but%20it%20doesn%E2%80%99t%20include%20the%20canvas%20straight%20away.%20You%20must%20add%20in%20canvas%20support%20so%20jsdom%20can%20find%20it.))

## Proof Of Work 📹

https://user-images.githubusercontent.com/14077479/179906952-b1715d5a-a1b0-4d6c-bcfe-ea9c6e8b34d9.mov

### CODE REVIEW

File: jest.setup.ts [hunk 1]
```diff
@@ -2,3 +2,4 @@
 // Learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
 import '@testing-library/jest-dom/extend-expect';
+import 'jest-canvas-mock';
```
Comment (hanam1ni) [inline jest.setup.ts:5]: Note to reviewer: Necessary for Jest to run a test on pages that contain canvas. [Reference](https://tnodes.medium.com/setting-up-jest-with-react-and-phaser-422b174ec87e#:~:text=That%20null%20is%20actually%20supposed%20to%20be%20the%20canvas%20element.%20Jest%20uses%20jsdom%20as%20a%20psuedo%20DOM%2C%20but%20it%20doesn%E2%80%99t%20include%20the%20canvas%20straight%20away.%20You%20must%20add%20in%20canvas%20support%20so%20jsdom%20can%20find%20it.)

File: src/pages/surveys/outro.page.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    setTimeout(() => {
+      dispatchAction({
+        type: ACTIONS.SET_CURRENT_SURVEY,
+        value: null,
+      });
+
+      router.push('/');
+    }, REDIRECT_AFTER_MS);
+  }, []);
```
Comment (malparty) [inline src/pages/surveys/outro.page.tsx:36]:
```suggestion
  }, [currentSurvey, dispatchAction]);
```
Should we add these 2 dependencies to the `useEffect` hook? 🤔  Or are these dependencies stateless? (I don't remember 100% for `useContext` values :P )

**Review by malparty — APPROVED** [review]
Just one minor comment, the rest LGTM :)

**Review by hoangmirs — APPROVED** [review]

File: src/pages/surveys/outro.page.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  const outroMessage =
+    currentSurvey?.outro?.text || 'Thanks for taking the survey.';
```
[the remark this answers is not in the document]
Comment (hanam1ni) [reply src/pages/surveys/outro.page.tsx:39]: I think we should use the same copy as the design
image

File: src/pages/surveys/outro.page.tsx (hunk 2 above)
Comment (hanam1ni) [reply src/pages/surveys/outro.page.tsx]: Updated in [51c4370](https://github.com/hanam1ni/react-surveyor/pull/94/commits/51c4370952ca8c214d23d0eed113ee984e23c7fe)

File: src/pages/surveys/outro.page.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (malparty) [reply src/pages/surveys/outro.page.tsx:39]: Yeah, it was a joke... I mean, in an IC you can take some freedom for things not related to "area to learn/practice" (like a label) ;-)

**Pull request merged**